oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/measure/BanyanDBMetadataQueryDAO.java

```diff
@@ -113,13 +114,14 @@ public List<ServiceInstance> listInstances(Duration duration, String serviceId)
         MeasureQueryResponse resp = query(InstanceTraffic.INDEX_NAME,
                 INSTANCE_TRAFFIC_TAGS,
                 Collections.emptySet(),
+                new TimestampRange(0, duration.getEndTimestamp()),
```

Comment:
What does this mean? @lujiajing1126 I feel this is not a good practice.

Reply:
Can we use `duration.getStartTimestamp` as the left boundary of the search range? @hanahmily

Reply:
OK, I think the time bucket is never updated since added. We have scan from 0 to end time condition.

Reply:
Exactly, we have to scan from 0